src/plots/geo/geo.js

```diff
@@ -153,8 +165,10 @@ proto.onceTopojsonIsLoaded = function(geoData, geoLayout) {
 
     var traceKeys = Object.keys(traceData);
     for(var j = 0; j < traceKeys.length; j++){
-        var traceKey = traceKeys[j];
-        Plots.getModule(traceKey).plot(this, traceData[traceKey], geoLayout);
+        var moduleData = traceData[traceKeys[j]];
+        var _module = moduleData[0]._module;
```

Comment:
@mdtusz sorry for sneaking this in here.

I believe grabbing `_module` from the trace objects is more robust then relying on `Plots`. That way, a plot's required trace modules are scoped inside each plot state, instead of being library-wide globals.